src/index.js

```diff
@@ -141,7 +141,7 @@ class Dayjs {
   }
 
   unix() {
-    return Math.floor(this.valueOf() / 1000)
+    return Math.floor(new Date(this.$d.toUTCString()) / 1000)
```

Comment:
I don't this is a proper fix. new date will always use local time zone

Reply:
Yes. I just realized this myself.